README.adoc

```diff
@@ -155,44 +142,54 @@ include::{common-includes}/gitclone.adoc[]
 
 == Starting and preparing your cluster for deployment
 
-To begin working with your Minikube cluster run the following command from the command line:
+Start up your {kube} cluster. If you are using Docker for Windows just start your Docker environment. If you are using Minikube then run the following command from the command line:
 
 ```
-# minikube with virtualbox
 minikube start
-# minikube with hyper-v
-minikube start --vm-driver hyperv --hyperv-virtual-switch "Primary Virtual Switch"
 ```
 
-This command sets up and starts a local {kube} cluster that is ready for deployment out of the box.
 Later, when you no longer need your cluster, you can stop it with `minikube stop` and delete it completely
 with `minikube delete`.
 
-Next, in order to setup an Ingress, you'll need to deploy the NGINX Ingress controller, which conveniently
-comes as an optional add-on in Minikube. To enable this add-on, run the following command:
+For any environment, validate that you have a healthy kubernetes environment by running the following command:
+```
+kubectl get nodes
+```
+
+This should return a `Ready` status for the master node. 
+
+Next, in order to setup an Ingress, you'll need to deploy the NGINX Ingress controller.
+If you are using Minikube this can be done through simply enabling an optional add-on in Minikube. To enable this add-on, run the following command:
 
 ```
 minikube addons enable ingress
 ```
 
-When enabled, Minikube deploys two Pods in the `kube-system` namespaces. The first Pod, called `default-http-backend`, 
+If you are not using Minikube then follow the platform-specific instructions at https://kubernetes.github.io/ingress-nginx/deploy/ . The Docker for Mac instructions are equally appropriate for Docker for Windows, where you should run the following two commands:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/mandatory.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/provider/cloud-generic.yaml
+``` 
+ 
+This results in the deployment of two pods. The first Pod, called `default-http-backend`, 
 is reponsible for routing invalid requests to a default HTTP backend. The second Pod, called `nginx-ingress-controller`, 
 runs the actual NGINX Ingress controller. Make sure that both of these Pods are in the ready state 
 before you proceed:
 
 [source, role="no_copy"]
 ----
-$ kubectl get pods --namespace kube-system
-NAME                                        READY     STATUS    RESTARTS   AGE
-default-http-backend-59868b7dd6-v4qh7       1/1       Running   0          11m
-nginx-ingress-controller-67956bf89d-6sg6r   1/1       Running   0          11m
+$ kubectl get pods --all-namespaces
```

Comment:
We typically don't have the command and output in the same box. Also we usually don't have an indicator for the prompt such as `$`.